custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_vacuum.yaml

```diff
@@ -9,6 +9,7 @@ card_vacuum:
     ulm_card_vacuum_name: "[[[ return entity.attributes.friendly_name ]]]"
     ulm_card_vacuum_icon: "[[[ return entity.attributes.icon ]]]"
     ulm_card_vacuum_room:
+    ulm_card_vacuum_room_icon: "mdi:sofa"
```

Comment:
I do agree with adding the option to change the icon manually. But I think de default should be the icon set in the GUI of HA. As this will now always be overridden by the default.
This can be achieved with 
```icon: "[[[ return entity.attributes.icon ]]]"``` or 
` icon: "[[[ return states[variables.ulm_card_vacuum_room].attributes.icon ]]]"`